core/new-gui/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
+
+describe('new-gui App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should display welcome message', () => {
+    page.navigateTo();
+    expect(page.getParagraphText()).toEqual('Welcome to texera!');
```

Comment:
I understand that this is just the first PR to integrate the testing mechanism, but we should settle on a proper naming and directory structure, so that tests can be mapped to components easily. Something like we have for backend

Reply:
This file is generated by Angular CLI as a placeholder of the file structure. So it's sort of official recommended placeholder test. We haven't integrated e2e test, which means "end to end test". into our project yet. You can ignore everything under `e2e`.

Reply:
NVM. Just saw that this is an automatically generated page. Individual component test files are attached with the components themselves. However, for the end to end tests too, we should formalize a directory structure.

Reply:
This directory structure is generated by Angular CLI, so we can view it as official recommended best practice directory structure.